SystemyCzasuRzeczywistego/threads.aadl

```diff
@@ -47,39 +47,39 @@ public
 			video_stream: in parameter Video_stream;
 			person_on_video_detected: out parameter BooleanValue;
 		properties
-			source_language => (Ada95);
-			source_name => "RoomController.VideoThread";
---			source_text => ("room.c");
+			source_language => (C);
+			source_name => "roomController_VideoThread";
+			source_text => ("room.c");
```

Comment:
Zmieniłem . na _ w nazwie funkcji, nie wiem czy z . by działało.

Reply:
Spoko :)
To było wymagane, bo piękny język `Ada` nie akceptował podkreśleń w nazwie